README.md

```diff
@@ -0,0 +1,67 @@
+# Winged Velociraptors
+
+A game based on One Night Ultimate Werewolf, but tailored specifically for our college friend group trying to attribute
+any card actions based on each individual's personality. 
+
+## Installs
+
+NPM Installs on Client side
+
+```
+npm install
+npm install antd 
+npm install react-router-dom
```

Comment:
Shouldn't these packages be in the `package.json`? Then they would automatically be installed with the `npm install` call.

Reply:
It was more for me to keep track of everything but yes you are correct